### PR TITLE
Make sure to set the path to the cert file using a windows path:

### PR DESCRIPTION
**Pull request by stickycode**
yarn expects a windows path not a unix path

### CODE REVIEW

File: branchout-yarn [hunk 1]
```diff
@@ -5,9 +5,14 @@ set -e
...
     if test -f "${PROJECTION_DIRECTORY}/.branchout/cacerts"; then
-      export NODE_EXTRA_CA_CERTS="${PROJECTION_DIRECTORY}/.branchout/cacerts"
+      if [ "$(command -v cygpath)" ]; then
+        NODE_EXTRA_CA_CERTS="$(cygpath --windows "${PROJECTION_DIRECTORY}/.branchout/cacerts")"
+      else
+        NODE_EXTRA_CA_CERTS="${PROJECTION_DIRECTORY}/.branchout/cacerts"
+      fi
 
       echo "NODE_EXTRA_CA_CERTS: ${NODE_EXTRA_CA_CERTS}"
+      export NODE_EXTRA_CA_CERTS=
```
Comment (fredcooke) [inline branchout-yarn:15]: Should this be down a line, after the fi or in an else block?

Comment (yunj) [reply branchout-yarn]: Thanks

**Review by yunj — APPROVED** [review]
Ship it!

File: branchout-yarn (hunk 1 above)
Comment (stickycode) [reply branchout-yarn]: good spotting

**Review by squirkenz — APPROVED** [review]

**Review by fredcooke — APPROVED** [review]
looks better now! :-) LGBT

**Comment by stickycode** [comment]
this was testing on a windows machine and mac machines, pending travis being fixed

travis just changed the payment plans and 'free' had to be reselected, working again now

**Comment by fredcooke** [comment]
Glad you figured that out! :-) I should get on with the next one!

**Comment by stickycode** [comment]
@fredcooke  yes please

**Pull request merged**